apps/python-docs/pages/agent-guidebook/required-skills.md

```diff
@@ -0,0 +1,33 @@
+<a id="blocks"></a>
+
+# Required Skills
+
+This guidebook is for **Python programmers**.
+
+We've written it assuming no prior experience building with Agents, Tools, or LLM apps. Though experience with these is of course helpful.
+
+A [Steamship API Key](/account/api) is all you'll need to do everything in the guidebook -- from DALL-E to Stable Diffusion to Voice Generation to Vector Search.
+
+## Note for workshops and classes
+
+Several workshops and classes have used this guidebook as curriculum.
+
+The biggest suggestion from those experiences is that **everyone begin by forming pairs** so that more-skilled programmers are evenly distributed around the room.
+
+Here is an activity you can use.
+
+    **Programmers!**, please put your laptops on your desk to identify yourself.
```

Comment:
Should this section be a react component? 
![Screenshot 2023-08-16 at 5 02 51 PM](https://github.com/steamship-core/steamship-frontend/assets/11764815/221afb57-b15d-418f-bceb-12bfe4cb4b60)

Reply:
Happy to make this a component, but curious to know why that's your instinct -- is the idea so that it could be reused?